content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,121 @@
+# Requirements
+
+When you have a model saved via MLEM and want to use it, the first step is to
+make sure you have the right dependencies and packages. MLEM can get the list of
+requirements for you automatically for some type of requirements.
```

Comment:
Don't quite get the sentence. Maybe something like "Given a model, MLEM can print its Python or Unix dependencies for you to install"?

Reply:
I don't want to be specific here about `Python or Unix` since we have different kind of requirements later.

Reply:
Then something like:
`Given a model, MLEM can get the list of requirements of different types (including Python, Unix, as well as some others).`
Otherwise it's not clear what is the "type of requirements"

Reply:
done